docs/content/manpages/geowave-addstore.txt

```diff
@@ -0,0 +1,31 @@
+//:geowave-addstore(1)
```

Comment:
Absolutely. I was wondering where all of those options were defined. I'll add that to the docs. Thanks.